ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -124,6 +125,18 @@
       csk/->snake_case
       (str "_" (UUID/randomUUID))))
 
+(defn- decode-query-in-concept
+  "Query can be URL encoded which will cause requests for provider=PROV1&options[spatial][or]=true
+         to fail. Extracts the Query, decodes it and then puts it back in."
+  [concept]
+  (if-let [metadata (json/parse-string (:metadata concept))]
+    (let [decoded-query (codec/form-decode-str (get metadata "Query"))
+          metadata-fixed (assoc metadata "Query" decoded-query)
+          metadata-as-str (json/generate-string metadata-fixed)
+          concept-fixed (assoc concept :metadata metadata-as-str)]
+      concept-fixed)
```

Comment:
You can use `as->` here and remove this let: https://clojuredocs.org/clojure.core/as-%3E

```suggestion
    (as-> metadata metadata
          (codec/form-decode-str (get metadata "Query")
          (assoc metadata "Query" metadata)
          (json/generate-string metadata)
          (assoc concept :metadata metadata-as-str))
```